docs/misc/deprecated.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "Docker Deprecated Features"
+description = "Deprecrated Features."
+keywords = ["docker, documentation, about, technology, deprecate"]
+[menu.main]
+parent = "mn_use_docker"
++++
+<![end-metadata]-->
+
+# Deprecated Features
+
+The following list of features are deprecated.
+
+### Old Command Line Options
+**Target For Removal In Release: v1.10**
```

Comment:
Can we add 

`(Deprecated from Release: v1.8.0)` (and possibly a date, or link to that version's release note)?